datastore/mocks/utils.go

```diff
@@ -73,7 +73,7 @@ func NewTxnWithMultistore(t *testing.T) *MultiStoreTxn {
 	result := &MultiStoreTxn{
 		Txn:             txn,
 		t:               t,
-		MockRootstore:   prepareRootStore(t),
+		MockRootstore:   prepareRootstore(t),
 		MockDatastore:   prepareDataStore(t),
```

Comment:
todo: Shouldn't the other `perpareXYZ` names change too?

Reply:
I'm changing the ones I touched. I'll work on converting the rest in a separate PR.